viewer/src/datamodels/pointcloud/PointCloudMetadataRepository.ts

```diff
@@ -9,11 +9,14 @@ import { PointCloudMetadata } from '@/datamodels/pointcloud/PointCloudMetadata';
 // TODO 25-05-2020 j-bjorne: Move into common type folder
 import { DataRepository } from '@/datamodels/cad/DataRepository';
 import { CadTransformationProvider } from '@/datamodels/cad/CadTransformationProvider';
+import { File3dFormat } from '@/utilities';
 
+type ModelIdentifierWithFormat<T> = T & { format: File3dFormat };
```

Comment:
It's copypasted 4 times... I'm not sure why do you need it at all, but If the `format` is something always needed then it's better to declare it somewhere like 

```ts
export class PointCloudMetadataRepository<TModelIdentifier extends { format: File3dFormat }>
```

Reply:
Extends won't do the trick here as we add the format field in the implementation of `loadData`. I don't really like the need for a format like this, but I didn't want to go down that path now...

Reply:
Ok, it's fine. I just pointed on what I noticed and didn't really understand.